README.md

```diff
@@ -14,28 +14,27 @@ The library provides a convenient API and its implementation is focused on maxim
 
 No additional third-party runtime dependencies are required to use this library.
 
-== Features
+## Features
 
-- Mask **strings** in any JSON structure that correspond to a configured target key (default)
-- Mask all values in **(nested) JSON arrays** that correspond to a configured target key (default)
-- Maks all values in **(nested) JSON objects** that correspond to a configured target key (default)
-- Mask **numbers** in any JSON structure that correspond to a configured target key (optional)
-- **Obfuscate** the original length of the masked value by using a fixed-length mask (optional)
-- Target key **case sensitivity configuration** (default: `false`)
-- **Block-list** (`masked`) or **allow-list** (`allowed`) interpretation of target key set (default: `masked`)
-- Masking valid JSON will always result in valid JSON
-- The implementation only supports JSON in UTF-8 character encoding
+* Mask ***strings*** in any JSON structure that correspond to a configured target key (default)
+* Mask all values in ***(nested) JSON arrays*** that correspond to a configured target key (default)
+* Maks all values in ***(nested) JSON objects*** that correspond to a configured target key (default)
+* Mask ***numbers*** in any JSON structure that correspond to a configured target key (optional)
+* ***Obfuscate*** the original length of the masked value by using a fixed-length mask (optional)
+* Target key ***case sensitivity configuration*** (default: `false`)
+* ***Block-list*** (`masked`) or ***allow-list*** (`allowed`) interpretation of target key set (default: `masked`)
```

Comment:
All these bold words are wrong now?

Reply:
Looks like they are ***bold cursive*** now 😄 I've changed it to just **bold**. Also found that image links we a bit wrong, fixed those too.